pmm/pmm2-release.groovy

```diff
@@ -36,8 +36,149 @@ pipeline {
+                                    fi
+                                    echo reprepro binary is \$(which reprepro)
+                                    pushd /srv/UPLOAD/${PATH_TO_CLIENT}/binary/debian
+                                        echo Looking for Debian build directories...
+                                        CODENAMES=\$(ls -1 | egrep -v 'cosmic|disco')
+                                        echo Distributions are: \${CODENAMES}
+                                    popd
+
+                                    #######################################
+                                    # source pushing, it's a bit specific #
+                                    #######################################
+
+                                    # pushing sources
+                                    if  [ -d /srv/UPLOAD/${PATH_TO_CLIENT}/source/debian ]; then
+                                        cd /srv/UPLOAD/${PATH_TO_CLIENT}/source/debian
+                                        DSC=\$(find . -type f -name '*.dsc')
+                                        for DSC_FILE in \${DSC}; do
+                                            echo DSC file is \${DSC_FILE}
+                                            for _codename in \${CODENAMES}; do
+                                                echo ===>DSC \$DSC_FILE
+                                                repopush --gpg-pass=${SIGN_PASSWORD} --package=\${DSC_FILE} --repo-path=\${REPOPATH} --component=main  --codename=\${_codename} --verbose || true
+                                                if [ -f \${REPOPATH}/db/lockfile ]; then
+                                                    sudo rm -vf \${REPOPATH}/db/lockfile
+                                                fi
+                                                sleep 1
+                                            done
+                                        done
+                                    fi
+
+                                    #######################################
+                                    # binary pushing                      #
+                                    #######################################
+                                    cd /srv/UPLOAD/${PATH_TO_CLIENT}/binary/debian
+
+                                    for _codename in \${CODENAMES}; do
+                                        pushd \${_codename}
+                                            DEBS=$(find . -type f -name '*.*deb' )
+                                            for _deb in \${DEBS}; do
+                                                repopush --gpg-pass=${SIGN_PASSWORD} --package=\${_deb} --repo-path=\${REPOPATH} --component=main --codename=\${_codename} --verbose
+                                            done
+                                        popd
+                                    done
+                                    #
+                                done
```

Comment:
please add `date +%s > /srv/repo-copy/version` as the last command before exiting ssh. it is going to be marker that we need to sync repos